### PR TITLE
Add all-namespace option

### PR DESCRIPTION
**Pull request by bgpat**
## Why

I want the feature to list secrets in all namespaces.

## What

Picked https://github.com/koudaiii/k8sec/commit/fc355a50cf5d47892be1ec9fb586bc245e18cc09.

### CODE REVIEW

File: cmd/list.go [hunk 1]
```diff
@@ -150,4 +156,5 @@ func init() {
 	RootCmd.AddCommand(listCmd)
 
 	listCmd.Flags().BoolVar(&listOpts.base64encode, "base64", false, "Show values as base64-encoded string")
+	listCmd.Flags().BoolVar(&listOpts.allnamespaces, "all-namespaces", false, "Show values in All Namespaces")
```
Comment (tomoasleep) [inline cmd/list.go]: How about adding short `-A` like the one of `kubectl get`？

>   -A, --all-namespaces=false: If present, list the requested object(s) across all namespaces. Namespace in current
context is ignored even if specified with --namespace.

Comment (bgpat) [reply cmd/list.go]: It's nice idea 👍

Comment (bgpat) [reply cmd/list.go]: ✅ https://github.com/wantedly/k8sec/pull/7/commits/247313899dfe60b2274fe628cf25478df9dcffa6

**Review by koudaiii — APPROVED** [review]

**Review by tomoasleep — APPROVED** [review]
👍

**Pull request merged**